src/main/java/PasswordGenerator.java

```diff
@@ -1,8 +1,84 @@
-/*
- * This Java source file was generated by the Gradle 'init' task.
- */
+import com.google.common.annotations.VisibleForTesting;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
 public class PasswordGenerator {
-    public String generatePassword() {
-        return "a+1";
+
+    public String generatePassword () {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Input minimum password length: ");
+
+        int minPassLength = scanner.nextInt();
+
+        System.out.println("Input how many special characters: ");
+
+        int characters = scanner.nextInt();
+
+        System.out.println("Input how many numbers: ");
+
+        int numbers = scanner.nextInt();
+
+
+        return runner(minPassLength, characters, numbers);
+
     }
+
+
+    public String runner ( int minPassLength, int numsOfSpecialCharacters, int numsOfNums ) {
```

Comment:
This `runner` method is weirdly named.  Feels like it should be called `generatePassword`?